site/content/api/Tenant-API.md

```diff
@@ -0,0 +1,264 @@
+
+The Tenant API is defined by means of AMQP 1.0 message exchanges, i.e. a client needs to connect to Hono using an AMQP 1.0 client in order to invoke operations of the API as described in the following sections.
+
+# Preconditions
+
+The preconditions for performing any of the operations are as follows:
+
+1. Client has established an AMQP connection with Hono.
+2. Client has established an AMQP link in role *sender* with Hono using target address `tenant`. This link is used by the client to send tenant commands to Hono.
+3. Client has established an AMQP link in role *receiver* with Hono using source address `tenant/${reply-to}` where *reply-to* may be any arbitrary string chosen by the client. This link is used by the client to receive responses to the tenant api requests it has sent to Hono. This link's source address is also referred to as the *reply-to* address for the request messages.
+
+This flow of messages is illustrated by the following sequence diagram (showing the AMQP performatives):
+
+![Tenant message flow preconditions](../tenant_ConnectToTenant.png)
+
+# Operations
+
+The operations described in the following sections can be used by clients to manage tenant information. In real world scenarios the provisioning of tenants will most likely be an orchestrated process spanning multiple components of which Hono will only be one.
+
+Conducting and orchestrating the overall provisioning process is not in scope of Hono. However, Hono's Tenant API can easily be used as part of such an overall provisioning process.
+
+
+## Add Tenant
+
+Clients use this command to initially *add* information about a new tenant to Hono.
+
+This operation is *optional*, implementors of this API may provide other means for adding tenant information, e.g. a RESTful API or a configuration file.
+
+**Message Flow**
+
+The following sequence diagram illustrates the flow of messages involved in a *Client* creating a tenant.
+
+![Add tenant message flow](../tenant_AddTenantSuccess.png)
+
+
+**Request Message Format**
+
+The following table provides an overview of the properties a client needs to set on an *add tenant* message in addition to the [Standard Request Properties]({{< relref "#standard-request-properties" >}}).
+
+| Name        | Mandatory | Location                 | Type     | Description |
+| :---------- | :-------: | :----------------------- | :------- | :---------- |
+| *subject*   | yes       | *properties*             | *string* | MUST be set to `add`. |
+
+The request message MUST contain a tenant payload as defined in the [Tenant Format]({{< relref "#tenant-format" >}}) section below.
+
+In the payload it is also possible to put additional custom meta information about the tenant (see the [Tenant Format]({{< relref "#tenant-format" >}}) section for further information).
```

Comment:
I think we can omit this sentence because it is already mentioned in the *Tenant Format* section to which you refer in the paragraph before already.

Reply:
removed